Movie/Movie/CinemaListScreen/Presenter/CinemaListScreenPresenter.swift

```diff
@@ -0,0 +1,44 @@
+// CinemaListScreenPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Презентер экрана со списков фильмов
+final class CinemaListScreenPresenter: CinemaListPresenterProtocol {
+    // MARK: - public properties
+
+    var worker: CinemaListNetworkWorkerProtocol!
```

Comment:
```suggestion
    var cinemaListNetworкService: CinemaListNetworкServiceProtocol?
```
worker в клин свифт у нас будут